assets/src/edit-story/components/library/panes/media/local/mediaPane.js

```diff
@@ -176,13 +209,27 @@ function MediaPane(props) {
   const onSelect = (mediaPickerEl) => {
     const resource = getResourceFromMediaPicker(mediaPickerEl);
     try {
+      if (
+        enableMediaPickerVideoOptimization &&
+        isFeatureEnabled &&
+        isTranscodingEnabled &&
+        !resource.local &&
```

Comment:
Is `resource.local` ever `true` here?

Reply:
I guess not. I guess I should remove it.

Reply:
```suggestion
```